src/Ast.ml

```diff
@@ -1563,7 +1644,7 @@ end = struct
       | Pexp_record (flds, _)
         when List.exists flds ~f:(fun (_, e0) -> e0 == exp) ->
           exposed_right_exp Non_apply exp
-          (* Non_apply is perhaps pessimistic *)
+      (* Non_apply is perhaps pessimistic *)
```

Comment:
I think if you revert this it will be preserved, and would be a better attachment

Reply:
done.